1.9/usage/pods/examples.md

```diff
@@ -4,8 +4,357 @@ feature_maturity: preview
 menu_order: 30
 ---
 
+# Annotated simple pod definition
+
+This pod, named `simple-pod` has a single container, `simpletask1`. The container pulls down an image (`python:3.5.2-alpine`) and runs a command.
+
+```
+{
+  "id": "/simple-pod",
+  "labels": {},
+  "environment": {},
+  "containers": [
+    {
+      "name": "simpletask1",
+      "exec": {
+        "command": {
+          "shell": "env && sleep 10000"
+        }
+      },
+      "resources": {
+        "cpus": 0.1,
+        "mem": 32
+      },
+      "image": {
+        "kind": "DOCKER",
+        "id": "python:3.5.2-alpine"
+      },
+      "artifacts": [],
+      "labels": {}
+    }
+  ],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ]
+}
+```
+
+## Basic pod fields
+
+| Field                 | Type    | Value                                                                                                                                                                                                             |
+|-----------------------|---------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id (required)         | string  | Unique ID for the pod.                                                                                                                                                                                            |
+| containers (required) | array   | See _Basic pod container fields_.                                                                                                                                                                                     |
+| volumes               | array   | All volumes associated with the pod.                                                                                                                                                                              |
+| volumes.name                | string  | Name of shared volume.                                                                                                                                                                                            |
+| volumes.host                | string  | Absolute path of the file or directory on the agent, or else the relative path of the directory in the executor's sandbox. Useful for mapping directories that exist on the agent or within the executor sandbox. |
+| networks              |         |                                                                                                                                                                                                                   |
```

Comment:
The value was missing on the doc I took this from. I believe that means that you can't just have a `networks` parameter with specifying the things below it (`mode`, etc). So there's no value to put in that field.